prod-aos-rcar-s4.yaml

```diff
@@ -44,7 +44,7 @@ common_data:
       url: "https://github.com/xen-troops/meta-xt-common.git"
       rev: master
     - type: git
-      url: "git@gitpct.epam.com:rec-inv/meta-aos-rcar-gen4.git"
+      url: "https://github.com/aoscloud/aos-core-rcar-gen4.git"
```

Comment:
meta-aos-rcar-gen4 and aos-core-rcar-gen4 are different repo. Should be meta-aos-rcar-gen4.